src/Functions/tokenizeChinese.cpp

```diff
@@ -0,0 +1,103 @@
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypeFixedString.h>
+#include <DataTypes/DataTypeArray.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnFixedString.h>
+#include <Columns/ColumnArray.h>
+#include <Common/Tokenizer/ChineseTokenizer.h>
+#include <Interpreters/Context_fwd.h>
+#include <Interpreters/ITokenExtractor.h>
+#include <Functions/IFunction.h>
+#include <Functions/FunctionHelpers.h>
+#include <Functions/FunctionFactory.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int BAD_ARGUMENTS;
+}
+
+class FunctionTokenizeChinese : public IFunction
+{
+public:
+    static constexpr auto name = "tokenizeChinese";
+
+    static FunctionPtr create(ContextPtr)
+    {
+        return std::make_shared<FunctionTokenizeChinese>();
+    }
+
+    String getName() const override { return name; }
+
+    size_t getNumberOfArguments() const override { return 1; }
+    bool isVariadic() const override { return false; }
```

Comment:
isVariadic = false is the default anyways (see `IFunction`). Let's remove l. 35.